source/documentation/team-dashboard.md

```diff
@@ -0,0 +1,72 @@
+# Team Dashboard
+
+We now have dashboards!
+
+We currently display dashboards on our TV monitors (two of them), running on ChromeOS. This section 
+outlines what's currently displayed on them.
+
+## Technical
+
+The top monitor has technical dashboards worth keeping an eye on throughout the day.
+
+### Grafana
+
+Grafana is an open source analytics and monitoring platform, and we use it for monitoring the health 
+of GovWifi in realtime.
+
+In the past, GovWifi used to be part of a shared organisation on Grafana. Now, GovWifi has its own
+organisation, and we have admin-level access so we can now make our own changes without having to 
+always involve RE. 
+
+We currently have two dashboards in Grafana:
+
+1. The first is for monitoring performance of GovWifi Product Pages and Tech Docs, and is available
+[here](https://grafana-paas.cloudapps.digital/d/KMxSG3DWk/govwifi?orgId=1&refresh=10s).
+
+2. The second is for monitoring GovWifi's SLIs for authentication journeys, as well as success rates
+for SMS and email responses, and is available
+[here](https://grafana-paas.cloudapps.digital/d/THPLfGxWk/govwifi-cloudwatch?orgId=4).
+
+Both of these are currently shown on the TV monitor.
+
+### Concourse
+
+Concourse is an open source CI/CD platform, and we use it for automating jobs for building, testing 
+and deploying software, as part of pipelines.
+
+Most of GovWifi's core pipelines are exposed (i.e. public, but read-only and without logs). These 
+are available [here](https://cd.gds-reliability.engineering/?search=team%3A%20govwifi), and are 
+shown on the TV monitor.
+
+It is useful to keep an eye on these, so that blocked pipelines can be spotted early and resolved.
+
+### Fourth Wall
+
+Fourth Wall is a GDS application for monitoring pull requests and their build statuses.
+
+We list all of our repositories on Fourth Wall on the TV monitor, as it helps in identifying 
+important and urgent pull requests that should be prioritised.
+
+### Google Analytics
+
+We currently have a tab with default Google Analytics dashboard open. This will be iterated over 
```

Comment:
the tab is a Datastudio dashboard powered by Google Analytics </pedantic>

Reply:
Not yet, but we could have that instead in the next iteration.